curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6400aa9c399ec53d971c368b.md

```diff
@@ -0,0 +1,162 @@
+---
+id: 6400aa9c399ec53d971c368b
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+It's now time to call the `displayAuthors` function. If you call the function just like that, you'll get all the authors populated in the UI. That's not what you want. You can just extract the number of authors you want. This is why we have the `startingIndex` and `endingIndex` variables.
+
+The best method for this use case is `slice()` array method. It lets you extract the portion of an array without changing the original array. It takes two arguments – the first is the starting index and the second is the ending index. Remember the `authorDataArr` is an array, so, you can use `splice()` on it.
+
+```js
+array.slice(startingPoint, endingPoint)
+```
+
+We need 8 authors at a time. So, call the `displayAuthors` function by using the `authorData` array and `slice()`. Use the `startingIndex` variable for the starting point and the `endingIndex` variable for the ending point.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here